apps/www/pages/company.tsx

```diff
@@ -247,7 +247,7 @@ const Investors = () => {
             <>
               <p>
                 <p className="text-lg">
-                  We've raised over $36 million in funding, backed by some of the world's leading
+                  We've raised over $116 M 🎉 in funding, backed by some of the world's leading
                   investors.
```

Comment:
I think we might be missing one of the tasks.

> Add Felicis to the logos

Reply:
@MildTomato I think that task was already covered and merged: https://github.com/supabase/supabase/pull/6878